infra/gcp/ensure-staging-storage.sh

```diff
@@ -217,16 +215,6 @@ for repo in "${RELEASE_STAGING_PROJECTS[@]}"; do
         color 6 "Empowering ${RELEASE_VIEWERS} as project viewers in ${PROJECT}"
         empower_group_as_viewer "${PROJECT}" "${RELEASE_VIEWERS}"
 
-        # TODO(justaugustus): Remove once the k8s-releng-prod GCP project is
```

Comment:
Just for reference...
Moving releng KMS was done as part of https://github.com/kubernetes/release/pull/1163.